client/css/style.css

```diff
@@ -2090,7 +2090,7 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
    For example: user has quit ( ===> L   O   L <=== )  */
 
 #windows .header .topic,
-#chat .message .text,
+#chat .message .text span,
```

Comment:
I don't like that, but I honestly couldn't find a decent class name. Suggestions?